src/sagemaker_containers/env.py

```diff
@@ -666,6 +667,64 @@ def model_server_workers(self):  # type: () -> int
         return self._model_server_workers
 
 
+class RequestEnv(Request, mapping.MappingMixin):
```

Comment:
this clearly models an HTTP request, so let's just call it Request.

Reply:
Yeah, I prefer Request as well. I will rename it.